src/java/org/apache/cassandra/db/CassandraKeyspaceWriteHandler.java

```diff
@@ -61,6 +64,43 @@ public WriteContext beginWrite(Mutation mutation, boolean makeDurable) throws Re
         }
     }
 
+    private CommitLogPosition addToCommitLog(Mutation mutation)
+    {
+        CommitLogPosition position;
```

Comment:
nit: Could just declare this below when it's assigned...or do away with it and return the result of `add()` directly.